black.py

```diff
@@ -246,7 +247,7 @@ def main(
     loop.add_signal_handler(signal.SIGTERM, cancel, _task_values)
     await asyncio.wait(tasks.values())
     cancelled = []
-    report = Report(check=not write_back, quiet=quiet)
+    report = Report(check=check, quiet=quiet)
```

Comment:
Just write `check=write_back is WriteBack.NO`

Reply:
Alright, I'll do it, I assume you mean `check=write_back is not WriteBack.NO`.

In my opinion, it is a little hacky and error-prone. I wouldn't be surprised if it breaks again in the future.